db/fetcher/indexer_iterators.go

```diff
@@ -39,6 +39,7 @@ const (
 	opNin   = "_nin"
 	opLike  = "_like"
 	opNlike = "_nlike"
+	opAny   = "_any"
```

Comment:
it's just there for composite indexes. We construct a slice of value matchers with every matcher being responsible for a corresponding field in the index to match. For some fields there might not be any criteria to match. For examples if you have composite index of /name/age/email/ and in the filter you specify only "name" and "email". Then the "_any" matcher will be used for "age".

I will add a comment to explain this.

Reply:
Okay cheers, I think I'm understanding this better.  In that case, can I also suggestion:

suggestion: Can those that are actual request filters reference those consts, and those which are not (i.e. opAny) remain as strings - this might make it clearer that some are special (and will also may it easier and less error prone to change the public filter syntax as these values will update when those do).  e.g.:
```
const (
   opEq = request.FilterOpEq,
   ...
   opAny   = "_any"
)
```
Although I found that `connor`/etc doesn't have a const yet for these, which would mean you have the hassle of changing all of those too, so feel very free to shrug this suggestion off as out of scope.